### DISCUSSION TITLE
Improvements to breadcrumb navigation paths

### DISCUSSION THREAD

**Discussion by lyswhut** [facebook/docusaurus#7605] [Ideas]
for examples:

![image](https://user-images.githubusercontent.com/20365169/173226623-25c0a2c3-5777-4375-bec3-05f11983c17b.png)

The full breadcrumb path for this page should be: `home > Docs > Introduction`
But now it is: `home > Introduction`

<br><br><br>

![image](https://user-images.githubusercontent.com/20365169/173226597-07aeff9c-b26e-4f0e-988e-0f6ddc451e64.png)

The full breadcrumb path for this page should be: `home > API > Client API`
But now it is: `home > Client API`

**Comment by slorber** [comment]
What you ask for doesn't make sense in all cases.

This is discussed and tracked here: https://github.com/facebook/docusaurus/issues/6953
